api/DOMMatrix.json

```diff
@@ -90,6 +90,103 @@
           }
         }
       },
+      "scale3dSelf": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMMatrix/scale3dSelf",
+          "description": "<code>scale3dSelf()</code>",
+          "support": {
+            "chrome": {
+              "version_added": "45"
+            },
+            "chrome_android": {
+              "version_added": "45"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "33",
+              "notes": "Starting in Firefox 69, the first parameter (<code>scale</code>) is now optional with a default value of 1, per the specification. Previously it was required."
+            },
+            "firefox_android": {
+              "version_added": "33",
+              "notes": "Firefox for Android requires the first parameter (<code>scale</code>)."
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": "32"
+            },
+            "opera_android": {
+              "version_added": "45"
+            },
+            "safari": {
+              "version_added": "11"
+            },
+            "safari_ios": {
+              "version_added": "11.3"
```

Comment:
This is odd, because I went through the code for Safari iOS specifically and was pretty sure that it was not present in 11. Not sure how I got that wrong... changing it anyway, but confused a bit.